.github/workflows/federated-integ-test.yml

```diff
@@ -0,0 +1,74 @@
+---
+
+name: Federated integ test
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+      - v1.*
+  pull_request:
+    branches:
+      - master
+      - v1.*
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  lte-integ-test:
+    runs-on: macos-10.15
+    steps:
+      - run: echo "::set-output name=date::$(date +'%m-%d-%Y--%H-%M-%S')"
+        id: date
+      - uses: actions/checkout@v2
+      - name: setup pyenv
+        uses: "gabrielfalcao/pyenv-action@v8"
+        with:
+          default: 3.8.5
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8.5'
+      - name: Install Docker
```

Comment:
Use this module to install docker and docker-compose : https://github.com/marketplace/actions/setup-docker